pkg/kdk/update.go

```diff
@@ -112,13 +121,13 @@ func Update(cfg *KdkEnvConfig) {
 	}
 
 	if needsUpdateImage(cfg) {
-		log.Info("Updating KDK image")
-		err := pullImage(cfg, cfg.ConfigFile.AppConfig.ImageRepository + ":" + latestReleaseVersion)
+		log.Info("Updating KDK docker image")
+		err := pullImage(cfg, cfg.ConfigFile.AppConfig.ImageRepository+":"+latestReleaseVersion)
```

Comment:
Why dropped the spaces here?  It's harder to read without them.

Reply:
gofmt wants to drop the whitespace.